Lib/logging/__init__.py

```diff
@@ -1094,6 +1096,8 @@ def emit(self, record):
             # issue 35046: merged two stream.writes into one.
             stream.write(msg + self.terminator)
             self.flush()
+        except RecursionError as e:  # See issue 36272
```

Comment:
Why not just `except RecursionError:` - why is the `as e` needed?

Reply:
This is a mistake, thanks.